project/Dependencies.scala

```diff
@@ -24,7 +24,6 @@ object Dependencies {
     // when updating config version, update links ActorSystem ScalaDoc to link to the updated version
     val config        = "com.typesafe"                % "config"                       % "1.2.1"       // ApacheV2
     val netty         = "io.netty"                    % "netty"                        % "3.8.0.Final" // ApacheV2
-    val protobuf      = "com.google.protobuf"         % "protobuf-java"                % "2.5.0"       // New BSD
```

Comment:
We don't only want it to be a binary compatible drop-in, it should also be a source and dependency compatible drop-in. 
Here we remove the transitive dependency that users may depend on having. Or have I missed something?